app/src/main/res/layout-land/content_main.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file was not there in your previous PR. Check if you can do it with one layout.

Reply:
I have removed this file. A separate xml is not required for landscape. I could do that using drawables only. Thanks for pointing it out.